### PR TITLE
[TorchToTcp] lowering support for aten.size.int op

### PR DESCRIPTION
**Pull request by zezhang**
Lowering `aten.size.int` op to `tensor::dim` op during torch-to-tcp.

Test (in docker):

`bazel test //test/...`

### CODE REVIEW

File: lib/Conversion/TorchToTcp/Misc.cpp [hunk 1]
```diff
@@ -133,6 +133,39 @@ class ConvertValueTensorLiteralOp
...
+    }
+    auto idxOp = rewriter.create<arith::ConstantIndexOp>(loc, idxVal);
+    auto dimOp = rewriter.create<tensor::DimOp>(loc, self, idxOp);
+    auto result =
+        rewriter.create<arith::IndexCastOp>(loc, rewriter.getI64Type(), dimOp);
+
+    rewriter.replaceOp(op, result);
+    if (constIntOp->hasOneUse()) {
+      rewriter.eraseOp(constIntOp);
+    }
```
Comment (navahgar) [inline lib/Conversion/TorchToTcp/Misc.cpp:163]: Why is this necessary? Shouldn't this be automatically taken care of by DCE pass (canonicalizer)?

File: test/Conversion/TorchToTcp/misc.mlir
Comment (navahgar) [inline test/Conversion/TorchToTcp/misc.mlir:1]: I'm assuming most the changes in this file (except the new test below) are just reformatting the existing tests.

File: lib/Conversion/TorchToTcp/Misc.cpp (hunk 1 above)
Comment (zezhang) [reply lib/Conversion/TorchToTcp/Misc.cpp:163]: Good point. Just removed.

File: test/Conversion/TorchToTcp/misc.mlir
Comment (zezhang) [reply test/Conversion/TorchToTcp/misc.mlir:1]: Yes, that's correct. Only added a new test for the aten.size.int case. Rest just fix the formatting.

**Review by navahgar — APPROVED** [review]
LGTM
